package.json

```diff
@@ -2,24 +2,23 @@
 	"name": "preact-custom-element",
 	"version": "4.4.0",
 	"description": "Wrap your component up as a custom element",
+	"source": "src/index.js",
+	"types": "dist/index.d.ts",
 	"main": "dist/preact-custom-element.js",
 	"module": "dist/preact-custom-element.esm.js",
-	"types": "dist/index.d.ts",
 	"unpkg": "dist/preact-custom-element.umd.js",
-	"source": "src/index.js",
 	"scripts": {
-		"prepare": "microbundle",
+		"build": "microbundle -f cjs,es,umd",
 		"lint": "eslint src/*.{js,jsx}",
 		"test": "wtr src/*.test.{js,jsx}",
-		"test:browsers": "wtr src/*.test.{js,jsx} --playwright --browsers chromium firefox webkit",
```

Comment:
Playwright is slow to install and rather unwieldy, and I don't think it's really needed here anyhow? Our tests aren't that interesting or on the bleeding edge. running in just chromium should be more than fine.